src/models/davObject.js

```diff
@@ -54,7 +54,7 @@ export class DavObject extends DAVEventListener {
 			_isDirty: false
 		});
 
-		this._exposeProperty('etag', NS.DAV, 'getetag');
+		this._exposeProperty('etag', NS.DAV, 'getetag', true);
```

Comment:
etag should not be publicly editable, just from inside this class.

Not a big deal though, let's just merge this and i will send a pull-request for this later.

Reply:
I added this for us to be able to specifically update an etag in case of errors :)
Otherwise you have to reload the entire app :/